NEWS.md

```diff
@@ -1,3 +1,7 @@
+# tune (development version)
+
+* The resampled confusion matrix is transposed when `conf_mat_resamped(tidy = FALSE)` is used (#372)
```

Comment:
```suggestion
* Fixed a bug where the resampled confusion matrix is transposed when `conf_mat_resamped(tidy = FALSE)` (#372)
```

I think it's a little unclear as is, like we are transposing on purpose? Which... you know 🙈